htsohm/material_files.py

```diff
@@ -383,31 +421,42 @@ def write_pseudo_atoms(file_name, atom_types):
         NOTE: ALL CHARGES ARE 0. IN THIS VERSION.
 
     """
+    temporary_charge = 0.
+
+    htsohm_dir = os.path.dirname(os.path.dirname(htsohm.__file__))
+    run_dir = os.path.join(htsohm_dir, run_id)
+    material_dir = os.path.join(run_dir, 'pseudo_materials')
+    file_path = os.path.join(material_dir, "%s.yaml" % uuid)
+    with open(file_path) as material_file:
+        material = yaml.load(material_file)
```

Comment:
Again, pass the pseudo_material file in, rather than load it? Also, we have the naming confusion here again between a material and pseudo_material